spec/features/visitor_views_neighborhood_homepage_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'visitor views neighborgood homepage', %Q(
```

Comment:
Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.